workshops/soundboard/README.md

```diff
@@ -0,0 +1,502 @@
+```
+
+There! Now our HTML file knows that it should grab the `main.js` file.
+
+For this workshop, we'll be using the handy library called jQuery inside our `main.js`. jQuery is a collection of functions and JavaScript code [fix] written to make writing JavaScript for the browser much easier. It provides a cleaner and simpler way to make things happen, almost like shortcuts.
+
+We'll let the browser know that we'll be using jQuery by once again using a `<script>` tag. jQuery can be found at [this URL](https://code.jquery.com/jquery-2.2.3.min.js), so that's what we will put as the value for `src`.
+
+Since we will be using jQuery in `main.js`, `main.js` will be reliant on jQuery. Thus, we will put the script tag containing jQuery above the one containing `main.js` in order to load jQuery before loading `main.js`.
+
+```html
+<script src="https://code.jquery.com/jquery-2.2.3.min.js"></script>
+<script src="main.js"></script>
+```
+
+Now we have successfully hooked up the JavaScript to the HTML. Our `index.html` now looks like this:
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Soundboard</title>
+</head>
+<body>
+  <script src="https://code.jquery.com/jquery-2.2.3.min.js"></script>
+  <script src="main.js"></script>
+</body>
+</html>
+```
+
+## Part II: User Interface
+
+Now that we've set up our HTML file, let's add some elements to the page to form the user interface of our soundboard.
+
+How should we design our user interface? This is what the user (you, or others) will be interacting with when trying to make sweet music. Since this soundboard will be on a webpage, we'll probably be clicking on things. One can click on many things, but the most inuitive is a button, which is what we will use.
+
+### Adding Buttons
+
+We can create buttons that imitate the buttons on a soundboard, by using [the button tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) (`<button>`).
+
+We use the button tag by typing the text we want on the button between the start (`<button>`) and close (`</button>`) tags.
+
+```html
+<button>Chop3</button>
+```
+
+This button will say "Chop3" on it. Let's make a few more.
+
+```html
+<button>Kick4</button>
+<button>Perc1</button>
+<button>Vox9</button>
+```
+
+You can have any text on your buttons that you desire, so long as you can still tell what sound they're going to make when you press them.
+
+Now `index.html` looks like this:
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Soundboard</title>
+</head>
+<body>
+  <button>Chop3</button>
+  <button>Kick4</button>
+  <button>Perc1</button>
+  <button>Vox9</button>
+  <script src="https://code.jquery.com/jquery-2.2.3.min.js"></script>
+  <script src="main.js"></script>
+</body>
+</html>
+```
+
+Now, as a user, we can tell the difference between each of the buttons, by what text is displayed on the buttons. But how can we differentiate them in code? One easy way is to supply each with an attribute known as an "`id`."
```

Comment:
inconsistency with the use of the term "id" (sometimes in `code` and sometimes not)